scripts/setup-pvs-proxy-rules

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+if [[ $# -ne 2 ]]
+then
+    echo "Usage: $0 [add|remove] <private_xs_path>"
+    exit 1;
+fi
+
+handle_error()
+{
+    logger -t setup-pvs-proxy-rules "Failed to read $1 from xenstore"
```

Comment:
I'd recommend to have a variable `LOG_TAG="setup-pvs-proxy-rules"` to use when calling `logger`. I'd also expect an error message on stderr, not just to syslog.